build-linux/build.sh

```diff
@@ -7,6 +7,7 @@ set -e
 export SCONS="scons -j${NUM_CORES} verbose=yes warnings=no progress=no"
 export OPTIONS="production=yes"
 export OPTIONS_MONO="module_mono_enabled=yes"
+export OPTIONS_ACKIT="accesskit_sdk_path=/root/accesskit/accesskit-c-0.15.1"
```

Comment:
Can probably be in the main `OPTIONS` like for macOS and Windows?

I'd also suggest renaming the folder to remove the version in the main script, so we only have one location to update when changing version.

Reply:
Added it separately since there's no arm Linux libs in official build, and I have not configured cross build for it.

> I'd also suggest renaming the folder to remove the version in the main script, so we only have one location to update when changing version.

Done.

Reply:
> Added it separately since there's no arm Linux libs in official build, and I have not configured cross build for it.

Added arm Linux build configs to the https://github.com/bruvzg/godot-accesskit-c-static and moved it `OPTIONS`.